lib/package-managers/npm.js

```diff
@@ -107,7 +107,19 @@ module.exports = {
             npm.commands.listAsync(args || [], true); // silent:true
     },
 
-    latest: cint.partialAt(view, 1, 'dist-tags.latest'),
+    latest: function (packageName, currentVersion, includePrerelease) {
+        if (includePrerelease) {
```

Comment:
This needs to work a little differently. It is possible that there is a higher number released just not on the latest tag. We don't want to accidentally pick that up. You should always try the latest tag first. If that ends up being a prerelease, then implement the logic to get the highest non-prerelease number instead (unless includePrerelease is specified).